utils/config.py

```diff
@@ -61,6 +61,18 @@ def get_deployer_account() -> Union[LocalAccount, Account]:
     return accounts.load(os.environ["DEPLOYER"]) if (is_live or "DEPLOYER" in os.environ) else accounts[4]
 
 
+def get_web3_storage_token() -> str:
+    is_live = get_is_live()
+    if is_live and "WEB3_STORAGE_TOKEN" not in os.environ:
+        raise EnvironmentError(
+            "Please set WEB3_STORAGE_TOKEN env variable to the web3.storage api token "
+            "to be able upload vote description to IPFS. You also could only calculate cid "
```

Comment:
maybe 'Please set WEB3_STORAGE_TOKEN env variable to the web3.storage API token to be able to upload the vote description to IPFS. Alternatively, you can only calculate cid without uploading to IPFS by calling `___` with `___`" (feel free to paraphrase)